cEP-0010.md

```diff
@@ -139,7 +142,17 @@ Here is the list of aspects to be implemented along with their corresponding
 - `Root.Formatting.Quotation`: This aspect checks on the quotations being used
   in your files given `Root.Formatting.Quotation.preferred_quotation`.
 - `Root.Formatting.Quotation.preferred_quotation`: This taste represents the
-  user's preferred quotation.
+  preferred quotation marks for the codebase. It ensures that every string
+  contains the selected style of quotes.
+  For example, `"This is double quoted"` is changed to `'This is double quoted'`
+  if `preferred_quotation` is `'`.
+- `Root.Formatting.Quotation.force_preferred_quotation`: This taste represents
```

Comment:
this taste should be called something like `escape_quotation` as that is what it does.

Also note that this problem area is much broader than simply escaping . See https://en.wikipedia.org/wiki/Delimiter_collision for how various languages deal with this.

IMO the entire quotation aspect needs to be redesigned, as its current design is fixated on features on Python, which is one of the least interesting languages in this regard.